ck.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/sh
+
+set -e
+
+function init() {
+	# Make sure submodule is registered and up-to-date.
+	git submodule update --init
+
+	# Fetch remote changes.
+	cd ckeditor-dev
+	# not needed git fetch
+
+	# Make sure our working copy is clean.
+	git reset --hard HEAD --quiet
+	git clean -fdx
+}
+
+function message() {
+	if [[ ! -z "$1" ]] ; then
+		echo
+		printf -- '-%.0s' {1..80}; echo ""
+		printf "%*s\n" $(( ( $(echo $1 | wc -c ) + 80 ) / 2 )) "$1"
+		printf -- '-%.0s' {1..80}; echo ""
+		echo
+
+		if [[ ! -z  "$2" ]] ; then
+			echo
+			echo "$2" | fold -w 80
+			echo
+		fi
+	fi
+}
+
+function usage() {
+	echo "Usage: ck.sh COMMAND"
+	echo
+	echo "  Where COMMAND is either:"
+	echo
+	echo "  🔧  setup: Setup everything to start working on a patch"
+	echo "  💉  patch: Generate patches "
+	echo "  🔥  build: Generate a patched version of CKEditor"
+	echo
+	echo
+}
+
+# Check arguments
+if [ $# -ne 1 ]; then
+	usage
+  exit 1
+fi
+
+COMMAND=$1
+
+case "$COMMAND" in
+	build)
+		VERSION=$2
+
+		init
```

Comment:
Given that `init` is destructive (it does a `git reset --hard` and a `git clean`), I think your "Are you sure?" prompt should actually happen before this.